addons/payment_stripe/data/payment_acquirer_data.xml

```diff
@@ -4,7 +4,7 @@
     <record id="payment.payment_acquirer_stripe" model="payment.acquirer">
         <field name="provider">stripe</field>
         <field name="support_authorization">False</field>
-        <field name="support_fees_computation">False</field>
+        <field name="support_fees_computation">True</field>
```

Comment:
Ideally, temporary test modifications should be in a separate commit with a clear title saying the commit shouldn't be merged ;).

Reply:
Thank you :)